lib/screens_web/views/v2/audio/normal_header_view.ex

```diff
@@ -1,11 +1,11 @@
 defmodule ScreensWeb.V2.Audio.NormalHeaderView do
   use ScreensWeb, :view
 
-  def render("_widget.ssml", %{text: text, branch: branch}) do
-    ~E|<p><s>This is the Green Line <%= branch %> branch to <%= text %></s></p>|
+  def render("_widget.ssml", %{audio_text: audio_text, branch: branch}) do
+    ~E|<p><s>This is the Green Line <%= branch %> branch to <%= audio_text %></s></p>|
   end
 
-  def render("_widget.ssml", %{text: text}) do
-    ~E|<p><s>This is <%= text %></s></p>|
+  def render("_widget.ssml", %{audio_text: audio_text}) do
```

Comment:
suggestion: `audio_text` seems redundant given the context; by definition all fields passed into this module are "audio" (coming from `audio_serialize/1`). It'd be reasonable to keep this as `text`.